package.json

```diff
@@ -64,6 +64,7 @@
     "babel-plugin-transform-imports": "2.0.0",
     "css-loader": "6.7.1",
     "eslint": "8.13.0",
+    "eslint-plugin-import": "^2.26.0",
```

Comment:
We to to prefer exact versions and let dependabot update versions one by one. So `"2.26.0"` without caret.

Reply:
Got it, fixed.